lib/cictl/solr_command.rb

```diff
@@ -0,0 +1,48 @@
+require "thor"
+
+require_relative "common"
+require_relative "solr_client"
+
+module CICTL
+  class SolrCommand < Thor
+    include Common
+
+    desc "commit", "Send a commit command to solr"
+
+    def commit
+      solr_client.commit!
+    end
+
+    desc "dump_deleted FILE", "Dump the solr docs marked as `deleted` to the given .jsonl file"
+    def dump_deleted(filename)
+      solr_client.dump_deletes_as_jsonl(filename)
+    end
+
+    desc "send_jsonl FILE", "Send pre-created records from a .jsonl(.gz.) file into solr"
+    option :batch_size, type: :numeric, desc: "Batch size when sending records", default: 1000
+
+    def send_jsonl(filename)
+      raise "'#{filename}' not found" unless File.exist?(filename)
```

Comment:
I would recommend using the built-in error class so bin/cictl catches it without a backtrace. Also worth using `readable?`:
```
raise CICTLError, "'#{filename}' not readable" unless File.readable?(filename)
```